src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -24,13 +24,14 @@ This separation of concerns means that the replication stream can operate withou
 
 The stream initialization proceeds as follows:
 
-1. The standby's consumer job connects via its system interface to the primary cluster and starts the primary cluster's physical stream producer job.
+1. The standby's consumer job connects via its system virtual cluster to the primary cluster and starts the primary cluster's physical stream producer job.
 1. The primary cluster chooses a timestamp at which to start the physical replication stream. Data on the primary is protected from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until it is replicated to the standby using a [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps).
 1. The primary cluster returns the timestamp and a [job ID]({% link {{ page.version.version }}/show-jobs.md %}#response) for the replication job.
 1. The standby cluster retrieves a list of all nodes in the primary cluster. It uses this list to distribute work across all nodes in the standby cluster.
 1. The initial scan runs on the primary and backfills all data from the primary virtual cluster as of the starting timestamp of the replication stream.
 1. Once the initial scan is complete, the primary then begins streaming all changes from the point of the starting timestamp.
 
+{% comment %}TODO Kathryn to update this graphic {% endcomment%}
```

Comment:
Could you make a JIRA issue for me to do this?

Reply:
https://cockroachlabs.atlassian.net/browse/DOC-9964